test/controller_test.exs

```diff
@@ -49,6 +49,12 @@ defmodule BlueBird.Test.ControllerTest do
     end
   end
 
+  describe "apigroup/2" do
+    test "generates a function api_group" do
+      assert Controller.api_group == %{name: "Bobtails", description: "The Bobtail Resource"}
```

Comment:
If we're going to use the `%{"name" => "description"}` format suggested above, die api_group macro should return a map in that format.

Reply:
This test was missing, I didn't change the `apigroup` macro. It would break at least `get_group_name/2` :-)